### ISSUE TITLE
Try to adapt the collective's "Budget" section for user profiles

### ISSUE THREAD

**Issue by Betree** [opencollective/opencollective#6130]
As part of https://github.com/opencollective/opencollective/issues/3749, we want to try and see if we can adapt the collective's budget section for user profiles both for consistency and because they make a better distinction between expenses and transactions.

At this stage, we want to keep this as a quick experiment to decide whether or not it is worth the effort.

Things that we can already foresee:
- We'll need to remove the statistics on the right column.

Things that we'll need to figure out:
- Does it works out of the box?
- Complexity of the migration?
- Do we want to make more improvements in the future, e.g. bring back the right column with different stats?

**Comment by Betree** [comment]
I experimented with this in https://github.com/opencollective/opencollective-frontend/pull/8436, but the result of this naive implementation doesn't look good to me:
- It doesn't help with the [issue](https://github.com/opencollective/opencollective/issues/3749) we're trying to resolve
- By default, it only show **received** expenses, which is not helpful for user profiles
- We're not showing any metrics, which is ok, but then the layout might not be ideal

Overall I'd be up to pursue the work on this issue; I think it's still a welcome change for consistency & simplicity if we can re-use the same section for both collectives and users. But I think it needs some @opencollective/design attention before we can roll it out for all profiles.

To test the demo: https://opencollective-frontend-git-labs-user-pro-e279fa-opencollective.vercel.app/betree#category-BUDGET (current version: https://staging.opencollective.com/betree#category-BUDGET).

**Comment by Betree** [comment]
cc @opencollective/design, you can try out the section with the link shared above. If you're happy with the result and can think about quick hacks to resolve the issues mentionned above we could try to implement something before the end of the sprint.

**Comment by Memo-Es** [comment]
@Betree, why there is a first section titled "Contributions", and then right after that another one titled "Recurring contributions"? Can we have only one section and get the recurring contributions management inside the settings?

**Comment by Betree** [comment]
> @Betree, why there is a first section titled "Contributions", and then right after that another one titled "Recurring contributions"? Can we have only one section and get the recurring contributions management inside the settings?

That's how it was designed a while ago, I haven't made any changes to that part. I agree that it does not look nice, but it's out of scope for this project/issue.
